routes/api/branches.js

```diff
@@ -1,8 +1,67 @@
 const express = require("express");
 const router = express.Router();
 
-router.get("/test", (req, res) =>
-  res.json({ msg: "This is the branches route" })
-);
+const Branch = require("../../models/Branch");
+const TextString = require('../../models/TextString');
+
+// Show
+router.get("/:id", (req, res) => {
+  Branch.findById(req.params.id)
+    .then((branch) => {
+      if (branch) {
+        res.json(branch);
+      } else {
+        res.status(404).json("No Such Branch");
+      }
+    })
+    .catch((err) => res.status(404).json(err));
+});
+
+// Index
```

Comment:
Remove those comments, they're not needed. It's really clear from the code what it does and we should always aim for a code that explains itself - comment only when it's crucial, and when it's not possible to understand the code otherwise :)